src/components/HomePage/AccountCard/AccountCardHeader/AccountCardHeader.tsx

```diff
@@ -45,18 +45,16 @@ const AccountCardHeader = ({
         <div className="funds">
           <div className="amount">
             <div
-              className={
-                type === NetworkType.MATIC ? 'matic-logo' : 'mana-logo'
-              }
+              className={network === Network.MATIC ? 'matic-logo' : 'mana-logo'}
             />
-            {amount}
+            {parseInt(amount.toFixed(0), 10).toLocaleString()}
           </div>
         </div>
         <div className="actions">
           <Button>
             <Icon name="plus" />
           </Button>
-          <Button onClick={() => onConvertToMaticMana()}>
+          <Button onClick={() => onConvert(network)}>
```

Comment:
Should we need to have 

```
handleConvert() {
      onReceive(network)
}

<Button onClick={handleConvert}>
```

I'm not sure as it is a one-liner if you prefer using a lambda